### ISSUE TITLE
Application Whitelist

### ISSUE THREAD

**Issue by GoogleCodeExporter** [soul4soul/peerblock#222]
```
Would be nice if users could "whitelist" certain applications as Trusted
Apps.  Of primary interest would be people who'd like to keep PeerBlock
entirely out of the way of their web browser, yet continue protecting the
rest of their system - this should drastically reduce the pain of running
with HTTP Blocked, something which you really should endeavor to do.
```

Original issue reported on code.google.com by `DarthBu...@gmail.com` on 6 Dec 2009 at 2:46
- Blocked on: #16

**Comment by GoogleCodeExporter** [comment]
Original comment by `peerbloc...@gmail.com` on 6 Dec 2009 at 2:48
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
I think it would be great if it could also set it so that peerblock only 
blocked 
certain apps and whitelist "everything else"
```

Original comment by `ejthill@gmail.com` on 6 Dec 2009 at 8:31

**Comment by GoogleCodeExporter** [comment]
```
@ejthill:
in other words, a blacklist.

so we could have a whitelist mode, where peerblock works on everything except 
for 
the exceptions you want
and a blacklist mode, where everything is allowed, except for a few programs 
you 
choose to apply the blocks on.
```

Original comment by `jimifloy...@gmail.com` on 8 Apr 2010 at 11:50

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Issue 361 has been merged into this issue.
```

Original comment by `peerbloc...@gmail.com` on 22 Nov 2010 at 12:55

**Comment by GoogleCodeExporter** [comment]
```
I have no plans on using peerblock again unless they add an application 
blacklist option. It is just too much trouble to set it up if all you want it 
for is to filter a few applications.
```

Original comment by `BigRedBr...@gmail.com` on 24 May 2013 at 5:11
